EssentialCSharp.Web/Views/Shared/_Layout.cshtml

```diff
@@ -164,230 +171,56 @@
                                 </ul>
                             </div>
                             <div v-cloak>
-                                Build: @if (ReleaseDateAttribute.GetReleaseDate() is DateTime date)
-                                {
-                                    <b>@TimeZoneInfo.ConvertTimeFromUtc(date, TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time")).ToString("d MMM, yyyy h:mm:ss tt", CultureInfo.InvariantCulture)</b>
-                                }
+                                <small>
+                                    Build: @if (ReleaseDateAttribute.GetReleaseDate() is DateTime date)
+                                    {
+                                        <b>@TimeZoneInfo.ConvertTimeFromUtc(date, TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time")).ToString("d MMM, yyyy h:mm:ss tt", CultureInfo.InvariantCulture)</b>
+                                    }
+                                </small>
                             </div>
                             <div v-cloak v-if="sidebarTab == 'search'" id="searchBar" class="search-window">
                                 <div class="input">
                                     <input type="text" />
                                     <button class="search-btn">
-                                        <i class="fa fa-search icon-color"></i>
+                                        <i class="fa fa-search icon-light"></i>
                                     </button>
                                 </div>
                             </div>
                         </div>
                     </Transition>
                 </div>
-                <div class="content-padding">
+                <div class="container mt-4 pt-2">
 
-                    <nav v-cloak class="page-nav-container">
-                        <div class="hover-text">
-                            <a id="prev-btn" :href="previousPageUrl" v-if="previousPageUrl" class="nav-btn"><i class="fa fa-solid fa-arrow-left icon-color nav-btn-icon"></i></a>
-                            <span class="tooltip-text tooltip-right">Previous Page (left-arrow)</span>
+                    <nav v-cloak class="turn-page">
+                        <div class="turn-page-tooltip">
+                            <a id="prev-btn" :href="previousPageUrl" v-if="previousPageUrl" class="arrow-btn"><i class="fa fa-solid fa-arrow-left icon-light arrow-icon"></i></a>
+                            <span class="turn-page-tooltip-text tooltip-right">Previous Page</span>
```

Comment:
I think some kind of indication that there are key bindings to activate these buttons might be helpful? 
![image](https://user-images.githubusercontent.com/22186029/225131394-e75b5252-a927-4c03-8dfb-8d94c6a6906d.png)